session/session.go

```diff
@@ -1384,6 +1371,9 @@ func (s *session) ParseWithParams(ctx context.Context, sql string, args ...inter
 		parseStartTime = time.Now()
 		stmts, warns, err = s.ParseSQL(ctx, sql, charsetInfo, collation)
 	}
+	if len(stmts) != 1 {
+		err = errors.New("run multiple statements internally is not supported:" + sql)
```

Comment:
I think it is better not to echo the full sql string back. If it uses user-supplied data, it could be used in other nefarious ways such as XSS on a webpage, leaking information etc.

Reply:
okay